utils.go

```diff
@@ -40,14 +40,22 @@ func VersionedRequestToProtoRequestWithShortIDs(block *builderSpec.VersionedSubm
 func ProtoRequestToVersionedRequest(block *SubmitBlockRequest) (*builderSpec.VersionedSubmitBlockRequest, error) {
 	switch consensusspec.DataVersion(block.Version) {
 	case consensusspec.DataVersionCapella:
+		blockRequest, err := ProtoRequestToCapellaRequest(block)
+		if err != nil {
+			return nil, err
```

Comment:
maybe return nil, errors.New("Error converting proto to cappella: "+ err.Error()")